Formula/ttfautohint.rb

```diff
@@ -21,22 +21,37 @@ class Ttfautohint < Formula
     depends_on "pkg-config" => :build
   end
 
+  option "with-qt", "Build ttfautohintGUI also"
+
+  deprecated_option "with-qt5" => "with-qt"
+
   depends_on "pkg-config" => :build
   depends_on "freetype"
   depends_on "harfbuzz"
   depends_on "libpng"
+  depends_on "qt" => :optional
 
   def install
+    args = %W[
+      --disable-dependency-tracking
+      --disable-silent-rules
+      --prefix=#{prefix}
+      --without-doc
+    ]
+
+    args << "--without-qt" if build.without? "qt"
+
+    ENV["QMAKESPEC"] = "macx-clang"
```

Comment:
I could export this variable only `if build.with? "qt"` but probably doesn't matter

Reply:
Thanks for testing!